test/Interop/Cxx/extern-var/extern-var-silgen.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift %s -I %S/Inputs -emit-sil -o %t/extern-var.sil -Xfrontend -enable-cxx-interop
+// RUN: %FileCheck < %t/extern-var.sil %s
+
+// CHECK: sil_global @counter : $Int32
+
+// CHECK: sil hidden @$s4main10getCounters5Int32VyF : $@convention(thin) () -> Int32
+// CHECK: %0 = global_addr @counter : $*Int32
+// CHECK: begin_access [read] [dynamic] %0 : $*Int32
```

Comment:
Please also include the actual load/store instructions.

Reply:
Done.